compaction/compaction_manager.cc

```diff
@@ -548,7 +545,7 @@ inline bool compaction_manager::maybe_stop_on_error(future<> f, stop_iteration w
         retry = false;
         do_stop();
     } catch (...) {
-        cmlog.error("compaction failed: {}: {}", std::current_exception(), decision_msg);
+        cmlog.error("compaction failed: {}: retrying", std::current_exception());
```

Comment:
the error msg is correct because will_stop overrides retry

Reply:
It is misleading to set `retry` to true in the following statement and return it later on.